packages/cyberstorm/src/components/PackageCard.tsx

```diff
@@ -0,0 +1,149 @@
+import React, { ReactNode } from "react";
+import styles from "./componentStyles/PackageCard.module.css";
+import { MetaItem } from "./MetaItem";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import {
+  faDownload,
+  faThumbsUp,
+  faHardDrive,
+} from "@fortawesome/free-solid-svg-icons";
+import { Tag } from "./Tag";
+
+const defaultImageSrc = "";
+
+export interface PackageCardProps {
+  packageName: string;
+  description?: string;
+  imageSrc?: string;
+  downloadCount: string;
+  likes: string;
+  size: string;
+  author?: string;
+  link: string;
+  lastUpdated?: string;
+  isPinned?: boolean;
+  isNsfw?: boolean;
+  isDeprecated?: boolean;
+  packageCardStyle: "default";
+  categories: string[];
+}
+
+/**
+ * Cyberstorm PackageCard component
+ */
+export const PackageCard: React.FC<PackageCardProps> = (props) => {
+  const {
+    packageName,
+    description,
+    imageSrc,
+    downloadCount,
+    likes,
+    size,
+    author,
+    link,
+    lastUpdated,
+    isPinned,
+    isNsfw,
+    isDeprecated,
+    packageCardStyle,
+    categories,
+  } = props;
+
+  const authorLink = ""; //TODO: author link
+
+  return (
+    <div
+      /* TS is not aware of defaultProps of function components. */
+      /* eslint-disable-next-line @typescript-eslint/no-non-null-assertion */
+      className={`${styles.root}
```

Comment:
Why is this prop split into two lines? I was ready to complain that you don't need to wrap the thing in string if there's just one value 😅

Reply:
Likely a remnant of having more stuff in there😆, made it prettier